### DISCUSSION TITLE
Achievements doesn't select the right repository.

### DISCUSSION THREAD

**Discussion by zxcodes** [community/community#27307] [Other Feature Feedback, Questions, & Ideas] [answered]
Hey there! I've noticed an issue with the Achievements feature that was launched recently. I have a few repositories with Stars. And, according to the achievement badge, the repository with many Stars will get the Badge, but in my case, a repository with more than 100 stars is ignored and the one with 17 stars got the badge. Is that how it works or I'm I missing something? Just curious.

**Comment by radar-blip** [comment]
@zxcodes, Sounds like a bug.  Here is some info on achievement badges:
https://github.com/Schweinepriester/github-profile-achievements#achievements

And here is where you can file a bug report (if need be): 
https://support.github.com/contact/bug-report

**Comment by airtower-luna** [comment, marked as answer]
AFAIK the *first* repository of yours to reach the required number of stars (e.g. 16) will be mentioned on the achievement. At least that's what the answer was in #20560. :smile_cat:

**Reply by radar-blip** [reply]
Yes correct, but it seems he already had a repo with more than 16. I could be wrong but thats what it sounds like.

**Reply by jfullstackdev** [reply]
> Yes correct, but it seems he already had a repo with more than 16. I could be wrong but thats what it sounds like.

it's not wrong, I call it threshold, the first repo ever to have 16 stars, for example, Linus Torvalds, the creator of Linux, 

![image](https://user-images.githubusercontent.com/47092464/181500188-f67419e8-c47b-46ce-a8b5-dd3db8d39c94.png)

but Linux has 135k stars as of this writing

**Comment by fernandounger** [comment]
It must be because your repository you earned the achievement got 16 stars before the other one, but when your other repository gets 128 stars it should show up in the achievement.
